romanesco/executors/pyspark.py

```diff
@@ -0,0 +1,30 @@
+import imp
+import json
+import sys
+from romanesco import spark
+
+
+def run(task, inputs, outputs, task_inputs, task_outputs, **kwargs):
+    custom = imp.new_module("custom")
+
+    sc = kwargs['_romanesco_spark_context']
+    custom.__dict__['sc'] = sc
+    custom.__dict__['spark_context'] = sc
```

Comment:
One other thing that might be handy to expose here is the spark module itself, since otherwise I imagine most everyone would have to include `import spark` in their script.